src/Microsoft.Health.Dicom.Functions/Update/UpdateDurableFunction.Orchestration.cs

```diff
@@ -131,6 +132,24 @@ public partial class UpdateDurableFunction
             {
                 logger.LogInformation("Update operation completed successfully");
             }
+
+            if (input.TotalNumberOfInstanceUpdated > 0)
```

Comment:
Don't we need use replaySafecounter?

Reply:
Good catch. I missed it while merging from main. Added it back.